app/middleware/index.js

```diff
@@ -5,8 +5,9 @@
  */
 const { json, urlencoded } = require('body-parser');
 const session = require('express-session');
-const { secret, environment } = require('../config').server;
+const cors = require('cors');
```

Comment:
This may be something we have to actually enable on the nginx side of things :thinking: